drafts/draft-ietf-ippm-ioam-ipv6-options.xml

```diff
@@ -156,7 +154,7 @@
          enabled per interface on every node within the IOAM domain.  Unless a
          particular interface is explicitly enabled (i.e., explicitly
          configured) for IOAM, a router MUST ignore IOAM Options.  As
-         additional security, IOAM domains SHOULD provide a mechanism to
+         additional security, IOAM domains MUST provide a mechanism to
 	 prevent injections at ingress or leaks at egress.</t>
```

Comment:
We seem to say that we MUST prevent injections at ingress - but IMHO we only want to prevent *non authorized* injections at ingress. Otherwise we'd prevent any IOAM encap at ingress.

Suggest that we 
s/prevent injections/prevent non-authorized injections/

Reply:
agreed